72_config_and_simplify_tls.md

```diff
@@ -0,0 +1,367 @@
+    the distribution already.
+  * `tpm_ownerpassword`: (Only Python agent)  TPM owner password (We do not
+    really need taking ownership for remote attestation). Will be probably
+    removed in the future.
+  * `extract_payload_zip`: Try to extract the payload
+  * `uuid`: UUID of the agent or one of the following options
+    * `generate`: creates random UUID
+    * `hash_ek`: Hash of the EK_pub in PEM format
+    * `hostname`: The hostname of the system
+    * `environment`: Uses the environment variable "KEYLIME_AGENT_UUID" as UUID.
+    * `dmidecode`: system UUID (Can we deprecate this option?)
+  * `enable_revocation_notifications`: Option to enable or disable listening for
+    revocation notifications from the verifier.
+  * `revocation_notifier_ip`: IP of the revocation notifier service
+  * `revocation_notifier_port`: Port of the revocation notifier service
+  * `revocation_cert`: Place of the certificate to check revocation messages against
+  * `revocation_actions`: List of revocations to run (Finally deprecate the Python only actions)
+  * `payload_script`: Name of the script to run
+  * `enable_insecure_payload`: Enable payloads even if mTLS is disabled
+  * `measure_payload_pcr`: Measure the payload into a PCR (With IMA is there any need for this feature anymore?)
+  * `exponential_backoff`: Use exponential backoff for retries
+  * `retry_interval`: Time to retry again or base for exponential retry if enabled
+  * `max_retries`: Maximum number of retries
+  * `tpm_hash_alg`: Hash algorithm used (e.g. for PCRs): sha1, sha256, sha384, sha512
+  * `tpm_encryption_alg`:  Used encryption algorithm: ecc, rsa (Has anyone tested actually ecc?)
+  * `tpm_signing_alg`: Signing algorithms
+  * `ek_handle`: Use persisted EK handle (Do we need this option?)
+  * `run_as`: Drop privileges to a different user after start
+* `verifier.conf`: Configuration of the verifier
+  * `ip`:  IP where the verifier listens on. (Should this be a list?)
+  * `port`: Port where the verifier listens on
+  * `registrar_ip`: IP of the registrar
+  * `registrar_port`: Port of the registrar
+  * `enable_agent_mtls`:  See TLS simplification
+  * `tls_dir`: See TLS simplification
+  * `server_key`: See TLS simplification
+  * `server_cert`: See TLS simplification
+  * `trusted_ca_in`: See TLS simplification
+  * `trusted_ca_out`: See TLS simplification
+  * `database_url`: Database configuration
+  * `database_pool_sz_ovfl`: TODO: Why do we need this option?
+  * `auto_migrate_db`: Automatically update DB schema using alembic
+  * `num_workers`: Number of processes (workers) to spawn
+  * `exponential_backoff`: Use exponential backoff for retries
+  * `retry_interval`: Time to retry again or base for exponential retry if enabled
+  * `max_retries`: Maximum number of retries
+  * `quote_interval`: polling interval for getting quotes from the agent
+  * `revocations`: Section for revocation related configuration
```

Comment:
This is really really good. It will open up the ability to add more revocation methods later.